content/400-reference/200-api-reference/300-environment-variables-reference.mdx

```diff
@@ -186,11 +186,7 @@ PRISMA_MIGRATION_ENGINE_BINARY=custom/my-migration-engine-unix
 PRISMA_INTROSPECTION_ENGINE_BINARY=custom/my-introspection-engine-unix
 ```
 
-<Admonition type="warning">
```

Comment:
Why remove this unrelated admonition about a different env var? The fmt change does not invalidate this.

Reply:
https://github.com/prisma/docs/pull/4502 fixes this